components/dialog/overview.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Overview
+page_title: Dialog Overview
+description: Overview of the Dialog for Blazor.
+slug: dialog-overview
+tags: telerik,blazor,dialog,overview
+published: True
+position: 0
+---
+
+# Dialog Overview
+
+The Dialog is a modal popup that brings information to the user.
```

Comment:
COnsider combining the first two paragraphs into one, as it is the first one is quite short and give rather little info